api/lql_test.go

```diff
@@ -225,6 +261,22 @@ func TestQueryGetQueryByIDOK(t *testing.T) {
 	assert.Nil(t, err)
 
 	assert.Equal(t, getExpected, getActual)
+
+	if strings.Contains(expectedResponseData, "queryLanguage") {
```

Comment:
If we are expecting a queryLanguage in the response, should we be validating that it's the right one?